web/src/components/njwds-extended/Alert.tsx

```diff
@@ -77,4 +79,8 @@ export const Alert = (props: Props): ReactElement => {
       </div>
     </div>
   );
-};
+});
+
+Alert.displayName = "Alert";
+
+export { Alert };
```

Comment:
Did this, because of some ESlint complaints due to using forwardRef (apparently its to make debugging easier)